src/test/java/hu/progmatic/swaglabs/LoginScreenTests.java

```diff
@@ -34,7 +34,7 @@ public void validUsernameEmptyPwdTest() throws InterruptedException {
         WebElement errorMessage = driver.findElement(By.cssSelector("h3[data-test='error']"));
 
         // annak ellenőrzése, hogy az elvárt hibaüzenetet tartalmazza-e az errorMessage html element tag-ek közötti értéke
-        Assert.assertEquals("Epic sadface: Password is required", errorMessage.getText());
+        Assert.assertEquals(errorMessage.getText(), "Epic sadface: Password is required");
```

Comment:
are you sure that is this a right order?

Reply:
Yes, checked the testng documentation and the first parameter is the actual result and the second one is the expected one. Believe me, my Friend!